observability/micrometer/src/test/resources/org/wildfly/extension/micrometer/micrometer-2.0.xml

```diff
@@ -0,0 +1,10 @@
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<subsystem xmlns="urn:wildfly:micrometer:2.0"
+    exposed-subsystems="*">
+    <otlp-registry endpoint="${test.endpoint:http\://localhost\:4318/v1/metrics}"
```

Comment:
When you add transformer tests it would be good to make sure that this xml can be transformed successfully